doc/admin-guide/files/records.config.en.rst

```diff
@@ -3061,6 +3061,7 @@ URL Remap Rules
 ===============
 
 .. ts:cv:: CONFIG proxy.config.url_remap.filename STRING remap.config
+   :deprecated:
```

Comment:
should this be done for the other vars as well?

Reply:
This was the only one with documentation :). I can add them though, but seems kinda lame to add new documentation for something we deprecate?

Reply:
I thought these were the equivalent to that one:

```
.. ts:cv:: CONFIG proxy.config.log.config.filename STRING logging.yaml
.. ts:cv:: CONFIG proxy.config.url_remap.filename STRING remap.config
.. ts:cv:: CONFIG proxy.config.ssl.server.multicert.filename STRING ssl_multicert.config
.. ts:cv:: CONFIG proxy.config.ssl.servername.filename STRING sni.yaml
```

Reply:
Hmmm, I must have missed those ... Let me mark them